crates/notion-core/src/distro/yarn.rs

```diff
@@ -36,13 +36,13 @@ pub struct YarnDistro {
     version: Version,
 }
 
-/// Check if the cached file is valid. It may have been corrupted or interrupted in the middle of
+/// Check if the fetched file is valid. It may have been corrupted or interrupted in the middle of
 /// downloading.
 // ISSUE(#134) - verify checksum
-fn cache_is_valid(cache_file: &PathBuf) -> bool {
-    if cache_file.is_file() {
-        if let Ok(file) = File::open(cache_file) {
-            match node_archive::load(file) {
+fn distro_is_valid(file: &PathBuf) -> bool {
+    if file.is_file() {
+        if let Ok(file) = File::open(file) {
+            match Tarball::load(file) {
```

Comment:
Just to double-check, the `node` versions are downloaded as either tarball (for Unix) or Zip (for Windows), while the `yarn` versions are just always tarballs, even on Windows?

Reply:
That's correct.